doc/stages/filters.griddecimation.rst

```diff
@@ -0,0 +1,45 @@
+.. _filters.crop:
+
+filters.crop
+============
+
+The **grid decimation filter** keeps only one point in a grid. The point selected could be the highest or the lowest point. It can be used to quickly filter vegetation points in order to keep only the canopy points.
+
+.. embed::
+
+Example
+-------
+This example transform highest points of classification 5 in classification 9, on a grid of 0.75m square.
```

Comment:
In the description,  it says keep only the highest or lowest points. Why are they being classified to `value` then?

Reply:
The description is not clear. My purpose is not to keep or delete points, but to transform (with the value) certains targeting points. We could, for exemple, add our own attribute who permits to select points for DSM (canopy for forest, lowest points for ground, etc...). I'm going to update the documentation.